packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -59,12 +44,7 @@ export interface IButtonSlots {
 /**
  * {@docCategory Button}
  */
-export interface IButton {
-  /**
-   * Sets focus to the Button.
-   */
-  focus: () => void;
-}
+export interface IButton extends IBaseButton {}
```

Comment:
I see `buttonRef` defined as a `React.RefObject<HTMLButtonElement>`, so I'm wondering if `IButton` is needed?

Reply:
This is an answer to this and the following comment.

I have `buttonRef` only as a view prop to use in state management but I don't want it to be exposed to users of `Button`. `IButton` is therefore still needed for the users to generate refs for the `componentRef` of the object. In this way, we can restrict unwanted manipulation of ref while still allowing devs to use the reference for things such as `focus`.

Reply:
Do we need to keep `IButton`? It's an empty interface so I'm wondering if it will be populated at some point.

Reply:
Oops, I just saw `IBaseButton`. Nevermind.

Reply:
I think I'll roll back those changes that I merged from the `BaseButton` change to not be dependent on the `Actionable` PR to close this as soon as possible.